.github/workflows/crowdin-download-and-merge.yml

```diff
@@ -0,0 +1,62 @@
+name: CrowdIn - Pull latest translations, verify and merge
+
+on:
+  # Allow manually triggering as well
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 1 * *' # Once a month (first of the month), at midnight
+
+permissions: write-all
+jobs:
+  crowdin-download-and-merge:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get current date
+        id: date
+        run: echo "current_date=$(date +'%d %b %Y')" >> $GITHUB_OUTPUT
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Synchronize with Crowdin
+        uses: crowdin/github-action@v1
+        with:
+          upload_sources: false
+          upload_translations: false
+          download_translations: true
+          skip_untranslated_files: true
+          localization_branch_name: l10n_crowdin_translations
+          export_only_approved: true
+          create_pull_request: true
+          pull_request_title: 'Crowdin translations for ${{ steps.date.outputs.current_date }}'
+          pull_request_body: 'Crowdin translations for ${{ steps.date.outputs.current_date }}'
+          commit_message: 'Crowdin translations for ${{ steps.date.outputs.current_date }}'
+          pull_request_base_branch_name: 'main'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID }}
+          CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_PERSONAL_TOKEN }}
+
+      - name: Run crowdin.py validation script
+        run: python crowdin.py
+        continue-on-error: false
+
+      - name: Find Pull Request
+        uses: juliangruber/find-pull-request-action@v1
+        id: find-pull-request
+        with:
+          branch: l10n_crowdin_translations
+
+      - run: echo "Pull Request ${number} (${sha})"
+        env:
+          number: ${{ steps.find-pull-request.outputs.number }}
+          sha: ${{ steps.find-pull-request.outputs.head-sha }}
+
+      - name: Merge Pull Request
+        uses: juliangruber/merge-pull-request-action@v1
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          number: ${{ steps.find-pull-request.outputs.number }}
+          method: squash
```

Comment:
You mean the E2E tests? Or the `crowdin.py` script? If it's the latter -> no, since it returns a non successful error code

Reply:
I mean our existing [CI tests](https://github.com/inaturalist/iNaturalistAndroid/blob/main/.github/workflows/CI.yml). Typically I would not merge manually if those failed, so that's how I'd like this to work too.

Reply:
FWIW, I'm not seeing a great way to achieve this, so maybe we ignore this problem. I thought there would be a way to configure juliangruber/merge-pull-request-action so it would only merge if the PR was ready, but it doesn't look like it. Then I thought maybe there was a way to use Github's auto-merge functionality, but it doesn't look like the Crowdin action supports enabling auto-merge.

Reply:
I think we might be able to do so, if we manage it on our own - basically using the GH client libraries (or directly calling the GH API) to do that. It's just a matter of prioritizing this

Reply:
Regarding making the entire workflow not merge in case the tests workflow fails - We could potentially do something [like this](https://stackoverflow.com/a/64733705/1233767) - however, we'll have to split this workflow into two:
1. Create a new PR with latest translations (that's it, without merging)
2. Let the tests workflow run
3. Have a new workflow that depends on the tests work run successfully - that new workflow will merge only if successful